### DISCUSSION TITLE
Reorder Presets

### DISCUSSION THREAD

**Discussion by euf0ria** [mainsail-crew/mainsail#2058] [General]
It would be nice to have the possibility to reorder the presets just like the macro section.  Look at example pictures below for further explanation.

![image](https://github.com/user-attachments/assets/13dada63-e608-43e1-9637-a559fa58de30)
"Bed 105" feels missplaced with human logic. I would rather see it below "Bed 75".

![image](https://github.com/user-attachments/assets/fb167d0e-0c63-427e-bbe9-bbcad2d64414)
Compare with macros that has handles for reordering.

**Comment by meteyou** [comment]
Thx for your Idea/FR. There exists already an open FR on the Mainsail repo: https://github.com/mainsail-crew/mainsail/issues/1728
Pls click / add an emoji to the first message. This will "upvote" this FR and get a higher priority. 

Another info to your presets: You don't need presets for "Bed" or "Nozzle". All your added preset temps for nozzle and bed are also in "heater bed" target dropdown. So if you set "Heater Bed -> 105" in your "Preset ABS/ASA", you "Heater Bed" target dropdown will also have this temp.

The Presets are sorted alphabetically. So just 6 is higher then 1. This is your main issue here... if you use "Bed 060", or just 2 spaces between, it will sorted "correct" (just as a "short fix" for now).

**Reply by euf0ria** [reply]
Those are great advices, thanx👍
